tools/topology/sof-apl-src-dmic.m4

```diff
@@ -0,0 +1,104 @@
+#
+# Topology for Apollo Lake with direct attach digital microphones array
+#
+
+# Capture configuration
+# DAI0 2ch 32b mic1-2
+# DAI1 2ch 32b mic1-2
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM6 <--- SRC <--- DMIC6 (DMIC01)
+# PCM7 <--- SRC <--- DMIC7 (DMIC16k)
+#
+
+dnl PIPELINE_PCM_DAI_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     period, priority, core,
+dnl     dai type, dai_index, dai format,
+dnl     dai periods, pcm_min_rate, pcm_max_rate,
+dnl     pipeline_rate, time_domain)
+
+# SRC capture pipeline 6 on PCM 6 using max channels 2.
+# Set 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_DAI_ADD(sof/pipe-src-capture.m4,
+	6, 6, 2, s32le,
+	1000, 0, 0, DMIC, 0, s32le, 3,
+	8000, 48000, 48000)
+
+# SRC capture pipeline 7 on PCM 7 using max channels 2.
+# Set 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_DAI_ADD(sof/pipe-src-capture.m4,
+	7, 7, 2, s32le,
+	1000, 0, 0, DMIC, 1, s32le, 3,
+	8000, 48000, 16000)
+
+#
+# DAIs configuration
+#
+
+dnl DAI_ADD(pipeline,
+dnl     pipe id, dai type, dai_index, dai_be,
+dnl     buffer, periods, format,
+dnl     deadline, priority, core, time_domain)
+
+# capture DAI is DMIC 0 using 3 periods
+# Buffers use s32le format, 1000us deadline on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-capture.m4,
+	6, DMIC, 0, NoCodec-6,
+	PIPELINE_SINK_6, 3, s32le,
+	1000, 0, 0)
```

Comment:
Please update to use `SCHEDULE_TIME_DOMAIN_TIMER`.
Master branch has been updated in the meantime.

Reply:
Thanks, done!